platform.h

```diff
@@ -123,6 +126,13 @@
 /* SPARC is big endian in general (but allows to access data in little endian too). */
 #define __FLASHROM_BIG_ENDIAN__ 1
 
+#elif IS_ARC
+#if defined(__arceb__)
```

Comment:
Is this documented somewhere? We couldn't figure out if it's really valid or if it's purely custom.

Looking at https://github.com/gcc-mirror/gcc/blob/master/gcc/config/arc/arc-c.c , it seems we should check if __BIG_ENDIAN__ is defined instead. For example:
```
#elif IS_ARC
#if defined(__BIG_ENDIAN__) 
#define __FLASHROM_BIG_ENDIAN__ 1
#else
#define __FLASHROM_LITTLE_ENDIAN__ 1
#endif
```